tests/test_cnes_sonar_scanner.py

```diff
@@ -0,0 +1,517 @@
+"""
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6252273)